src/commands/ping.ts

```diff
@@ -0,0 +1,10 @@
+import { SlashCommandBuilder } from 'discord.js';
+import client from '../index';
+
+export const info = new SlashCommandBuilder()
+  .setName('ping')
+  .setDescription('Tells you the bot\'s ping (ms)');
+
+export async function run(interaction: any) {
+  await interaction.reply(`${client.ws.ping}ms`);
```

Comment:
Use `interaction.client` instead of importing `client`.
```suggestion
  await interaction.reply(`${interaction.client.ws.ping}ms`);
```